haiku-apps/feedgator/feedgator-2.1.b.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Feed the Gator"
+DESCRIPTION="FeedGator is a feed/RSS aggregator. \
+It's a native python3 program that uses Haiku APIs through Haiku-PyAPI \
+wrapper and the feedparser module. \
+Check the settings to enable some features like automated feed checks \
+or launch minimized. \
+The code is released under MIT license."
+HOMEPAGE="https://github.com/tmtfx/feedgator"
+COPYRIGHT="2024 Fabio Tomat (TmTFx)"
+LICENSE="MIT"
+REVISION="2"
+srcGitRev="2.1.b"
+SOURCE_URI="https://github.com/tmtfx/FeedGator/archive/refs/tags/2.1.b.tar.gz"
+CHECKSUM_SHA256="7b3f7711ab6c9c7318357bf155fb3f6a25b75dcc29ec4b2b2f1a1e336a7ca0d1"
+SOURCE_DIR="FeedGator-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	feedgator = $portVersion
+	app:FeedGator = $portVersion
+	cmd:feedgator.py
+	"
+
+REQUIRES="
+	haiku
+	haiku_pyapi
+	feedparser
+	"
+PYTHON_PACKAGES+=(python310)
+PYTHON_VERSIONS+=(3.10)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	REQUIRES+="
+		haiku_pyapi_$pythonPackage
+		feedparser_$pythonPackage
+		"
+done
```

Comment:
When using only one Python version, there is no need for a loop like this. Just add `haiku_pyapi_python310` and `feedparser_python310` to `REQUIRES` directly, instead of the existing ones (which are empty anyway). You can put the `python310` part in a `pythonPackage` variable also.

Reply:
I'm going to change the recipe then.